executor/analyze.go

```diff
@@ -112,6 +113,14 @@ type analyzeTask struct {
 }
 
 func (e *AnalyzeExec) analyzeWorker(taskCh <-chan *analyzeTask, resultCh chan<- statistics.AnalyzeResult) {
+	defer func() {
```

Comment:
How about moving it to `Next` function? if recover it here, the `Next` will hang forever to waiting for the results.

Reply:
If it's in `Next`, `recover()` could not work, because `recover` must be inside some call stack of the panic.
`Next` and the `go analyzeWorker` are in different call stack.